### PR TITLE
feature/176: Added skip property

### LINKED ISSUE #176: Add skip property
Add a property to skip project-keeper.
Basically all maven plugins have such a property.

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #176

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,6 +181,8 @@ In addition this plugin can also fix the project structure. For that use:
 mvn project-keeper:fix
 ```
 
+You can skip the execution of project-keeper by adding `-Dproject-keper.skip=true` to your maven command.
```
Comment (morazow) [inline README.md:184]:
```suggestion
You can skip the execution of project-keeper by adding `-Dproject-keeper.skip=true` to your maven command.
```

File: pom.xml [hunk 2]
```diff
@@ -77,7 +77,7 @@
         <dependency>
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
-            <version>2.3.1</version>
+            <version>2.4.0-b180830.0359</version>
```
Comment (morazow) [inline pom.xml:80]: Is this correct release candidate version? It is very strange versioning number.

**Review by morazow — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (jakobbraun) [reply pom.xml:80]: You're right. Seems to be a pre-release

Comment (jakobbraun) [reply pom.xml:80]:
```suggestion
            <version>2.3.1</version>
```

File: doc/changes/changes_1.2.0.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Features
+
+* #176: Added skip property
+
+## Dependency Updates
+
+### Compile Dependency Updates
+
+* Updated `javax.xml.bind:jaxb-api:2.3.1` to `2.4.0-b180830.0359`
```
Comment (morazow) [inline doc/changes/changes_1.2.0.md:17]: Project keeper may fail on this :)) since it is different now.

**Review by morazow — APPROVED** [review]

**Pull request merged**